README.md

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 HUSH 1.0.9
+=======
+Zcash 1.0.12
+>>>>>>> b86b9feb6c3c1727cc3992da2f95c4a4f09a982b
```

Comment:
Looks like you committed some merge conflicts

Reply:
Okay, I think I see what you are talking about. Each of the files that were conflicted I added with git add and then committed. So the files that are in conflict are seem to have a duplication of some code that should not be there.  Tomorrow, I will hopefully correct this. I will open each conflicted file and take a gander at its contents to see the difference between the changes, then correct the conflicts.